deploy/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Builder image
+FROM golang:1.16 as builder
+
+# Workaround because migration tool is private repo
+ARG SSH_PRIVATE_KEY
+RUN mkdir /root/.ssh/ && \
+    echo "${SSH_PRIVATE_KEY}" > /root/.ssh/id_rsa && \
+    chmod 600 /root/.ssh/id_rsa && \
+    ssh-keyscan -t rsa -H github.com > ~/.ssh/known_hosts
+
+ENV GOPRIVATE=github.com/SAP/sap-btp-service-operator-migration
+RUN git config --global url.git@github.com:.insteadOf https://github.com/
+
+# TODO: sent changes to upstream
+# https://github.com/wozniakjan/sap-btp-service-operator-migration/pull/new/allow_in_cluster
+RUN git clone --branch=allow_in_cluster --depth=1 https://github.com/wozniakjan/sap-btp-service-operator-migration && \
```

Comment:
inCluster kubeconfigs are not supported by migration tool at the moment but I have a patch to allow that